.github/workflows/push.yml

```diff
@@ -1,6 +1,24 @@
 name: master
 on: push
 jobs:
+  prepare_conda_windows: # job is added in order to fix #1908
+    name: prepare_conda_windows
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["windows-latest"]
+        python-version: ["3.6"]
```

Comment:
If we are limiting to only one version of Python, I think we should pick the latest release.